tools/codegen/gen_backend_stubs.py

```diff
@@ -233,16 +233,7 @@ def make_file_manager(install_dir: str) -> FileManager:
                 'DispatchKey': dispatch_key,
                 'dispatch_namespace': dispatch_key.lower(),
                 'dispatch_helpers': dest.gen_registration_helpers(backend_indices[dispatch_key]),
-                'dispatch_namespaced_definitions': list(concatMap(
-                    dest.RegisterDispatchKey(
-                        backend_indices[dispatch_key],
-                        Target.NAMESPACED_DEFINITION,
-                        selector,
-                        rocm=False,
-                        cpp_namespace=cpp_namespace,
-                        class_method_name=f'{backend_dispatch_key}NativeFunctions'),
-                    grouped_native_functions
-                )),
+                'dispatch_namespaced_definitions': '',
```

Comment:
can you add a light comment - external backends don't need namespaced definitions now, but we can always add them back later.